roles/gpg/files/gpg-agent.conf

```
@@ -3,3 +3,6 @@
 # Cache passphrase 
 default-cache-ttl 21600 # Default to 6h
 max-cache-ttl 43200 # Maximum to 12h
+
+# Use Pinentry in GPG configuration
+pinentry-program /opt/homebrew/bin/pinentry-mac
```

en installant pinentry-mac, il m'indique plutôt le path suivant:

```suggestion
pinentry-program /usr/local/bin/pinentry-mac
```

---

Ah, je viens de vois que c'est une différence entre Mac Intel et ARM